src/components/LightUpChinatown/LightUpChinatownPage.tsx

```diff
@@ -68,6 +72,20 @@ const LightUpChinatownPage = () => {
           <CampaignInfoText color={'#CF6E8A'}>
             {daysUntilEnd} {t('lightUpChinatown.campaignDaysLeft')}
           </CampaignInfoText>
+        </TextContainer>
```

Comment:
This is from @danthaman44's PR right? Not sure what your rebase process is stuff like this keeps coming in 😂 but it's alright ahha